covasim/population.py

```diff
@@ -19,7 +19,7 @@
            'make_synthpop']
 
 
-class People(list):
+class People:
```

Comment:
Why not just inherit from `np.ndarray` and then I think most of these definitions wouldn't be necessary?

Reply:
I've tried and had some errors, but looking now I think this should be possible, let me try tomorrow